x/gov/client/utils/query.go

```diff
@@ -378,28 +277,10 @@ func QueryDepositByTxQuery(clientCtx client.Context, params v1.QueryDepositParam
 	return nil, fmt.Errorf("address '%s' did not deposit to proposalID %d", params.Depositor, params.ProposalID)
 }
 
-// QueryProposerByTxQuery will query for a proposer of a governance proposal by
-// ID.
+// QueryProposerByTxQuery will query for a proposer of a governance proposal by ID.
 func QueryProposerByTxQuery(clientCtx client.Context, proposalID uint64) (Proposer, error) {
```

Comment:
I will replace this one by a query in a follow-up, given that we store the proposer in the proposal.